ui/app/templates/components/clients/usage-stats.hbs

```diff
@@ -19,10 +19,11 @@
   <div class="columns">
     <div class="column">
       <StatText
+        class="column"
         @label="Total clients"
         @value={{@totalUsageCounts.clients}}
         @size="l"
-        @subText="The sum of entity clientIDs and non-entity clientIDs; this is Vault’s primary billing metric."
+        @subText="The sum of the clients listed; this is Vault’s primary billing metric."
```

Comment:
Not sure about this copy and would be happy to hear an alternative.

Reply:
I'd probably just say `sum of clients` or pull the subtext from the running client total stat above, but change to month "The number of clients which interacted with Vault during this month."

Reply:
Sounds good to me I'll update to the latter. Thanks!